### DISCUSSION TITLE
av1 video lagging

### DISCUSSION THREAD

**Discussion by nakyu24k** [mpv-player/mpv#15100] [Q&A]
I think im using a wrong library or something, im not sure how to fix this so i humbly request help, heres a clip https://streamable.com/k0rjej
heres what my mpv install looks like https://imgur.com/a/dAkmlvY, i have ffmpeg in path
my specs: r7 5800x, 80gb ram, rx7800xt win10 ltsc, many thanks

**Comment by llyyr** [comment]
https://github.com/mpv-player/mpv/issues/new/choose follow the instructions and fill out the template for a bug report
